recipes/c-progressbar/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "c-progressbar" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "38715897aaff04c87207c1c66c7d07cec6909d6930549c38286cbb34dd2a9fed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/limix/progressbar/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python  # [win]
+    - conda-build
+    - toolchain
+    - cmake
+    - ncurses
+  run:
+    - ncurses
```

Comment:
Please pin build and run time dependencies according to the [pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/d2cd7bc2c8bbfc83e7cbc11a32379fb9b77c0883/scripts/pin_the_slow_way.py#L40-L81 ).